Sources/Sentry/SentryEnvelope.m

```diff
@@ -165,17 +165,15 @@ - (instancetype)initWithUserFeedback:(SentryUserFeedback *)userFeedback
 - (_Nullable instancetype)initWithAttachment:(SentryAttachment *)attachment
                            maxAttachmentSize:(NSUInteger)maxAttachmentSize
 {
-    NSUInteger maxAttachmentSizeInBytes = 1024 * 1024 * maxAttachmentSize;
-
     NSData *data = nil;
     if (nil != attachment.data) {
-        if (attachment.data.length > maxAttachmentSizeInBytes) {
+        if (attachment.data.length > maxAttachmentSize) {
```

Comment:
L: What about having the unit in the name? `maxAttachmentSizeBytes`

Reply:
Makes sense I'm going to change it for Java as well 👏🏼

Reply:
Now that I think about it again, it is not really typical to suffix the size with bytes. For example, the [fileSize attribute](https://developer.apple.com/documentation/foundation/nsdictionary/1413465-filesize?language=objc) doesn't contain bytes in the name, neither does [`File.length`](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#length()) in Java.